examples/ads.amp.html

```diff
@@ -1602,6 +1603,18 @@ <h2>RUNative</h2>
       data-title="From the Web">
   </amp-embed>
 
+  <h2>SAS CI 360 Match</h2>
+  <amp-ad
+      width="468"
+      height="60"
+      type="sas"
```

Comment:
Missing the 'layout' attribute 😄

Reply:
@torch2424 okay, added the layout attribute to the example and the documentation. Hopefully we all good now.